deployments/porch/5-rbac.yaml

```diff
@@ -22,7 +22,7 @@ rules:
     verbs: ["get", "watch", "list"]
   - apiGroups: ["admissionregistration.k8s.io"]
     resources:
-      ["mutatingwebhookconfigurations", "validatingwebhookconfigurations"]
+      ["mutatingwebhookconfigurations", "validatingwebhookconfigurations", "validatingadmissionpolicies", "validatingadmissionpolicybindings"]
```

Comment:
Why are these needed? Is the porch server creating these?

Reply:
With the upgrade to the controller-runtime client it created what I though was a lot of noise regarding missing permissions but it turned out these are required to satisfy the new client.
All of the these rbac entries will need to be revisited and refined going forward.

Reply:
That's really not good. I think it needs an other fix. Allowing access to those resources are high-privileged, even noted in the k8s docs "Create with care" https://kubernetes.io/docs/reference/access-authn-authz/validating-admission-policy/#getting-started-with-validating-admission-policy

Reply:
Removed for now.
Feature added as GA in k8s v1.30.
Will need to revisit when base platform is upgraded in CI/test-infra